src/tiler/server.js

```diff
@@ -164,6 +168,26 @@ var config = {
                 req.params.sql = getSqlForStreamByReq(req);
             }
 
+            if (tableId.indexOf('drb_catchment') >= 0) {
+                switch (tableId) {
```

Comment:
Consider removing this to its own function like `getSqlForStreamsByReq` to keep this function small